setup.py

```diff
@@ -37,7 +37,7 @@ def get_static_files(path):
 with open('README.rst') as fobj:
     long_description = "\n\n" + readme_note + "\n\n" + fobj.read()
 
-install_requires = ['python-dateutil>=2.7.5,<3']
+install_requires = ['python-dateutil>=2.7.5,<3', 'tenacity>=6.3.0']
```

Comment:
I think tacking on `,<7` would be a wise precaution in case of future incompatible changes in tenacity.

Reply:
Thank you for your suggestion. I added `<7` on here and `tox.ini`.